### PR TITLE
feat: linkbutton with linknext

### PR DESCRIPTION
**Pull request by femacedogomes**
in this branch i created a link component based on Next/Link HOC.
task: https://www.notion.so/028386dd8cc04ffeaa06931d912bd0fc?v=b8d42189aed746d8a0a716c781c8d731&p=1b5a1e5dce1a46bfb27c3fbced3374a7&pm=s

### CODE REVIEW

File: src/components/atoms/Link/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  style?: 'primary' | 'white' | 'gray';
+};
+
+const LinkButton: React.FC<LinkProps> = ({ href, title, style }) => {
+  return (
+    <div>
+      <Link href={href} passHref>
+        <a
+          className={cx('text-end text-xs hover:underline ', {
+            ' text-scrummyOrange-500 decoration-scrummyOrange-500 ':
```
Comment (pedrohbl03) [inline src/components/atoms/Link/index.tsx:28]: i invite you to think this using other props;

**example:** bgColor, text Color, size, etc...

if we use a unique prop called styled, this component has a very coupled style.

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
see this to better practice in dumb components;

**Review by renanpontez — APPROVED** [review]

File: src/components/atoms/Link/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export type LinkProps = {
+  href: string;
+  title?: string;
+  bgColor?: 'none' | 'primary' | 'white' | 'gray';
+  textSize?: 'small' | 'normal' | 'large';
+  textColor?: 'primary' | 'white' | 'gray';
+};
+
+const LinkButton: React.FC<LinkProps> = ({
+  href,
```
Comment (pedrohbl03) [inline src/components/atoms/Link/index.tsx:15]: it's good add more props.
Example `externalLink` and if true, add a external icon and target blank in `<a>` tag.

**Review by pedrohbl03 — APPROVED** [review]
GOOOOOOD!

**Pull request merged**